daemon/graphdriver/quota/projectquota.go

```diff
@@ -194,6 +221,24 @@ func setProjectQuota(backingFsBlockDev string, projectID uint32, quota Quota) er
 	return nil
 }
 
+// getQuotaStat - get the quota stat
+// return 2 means quota is on
+func getQuotaStat(backingFsBlockDev string) (int, error) {
```

Comment:
Yes, it will still return 2. 2 means quota is enforced,  but root user can bypass enforcement and that is exactly what happens with overlayfs over ext4 since kernel v4.8, because overlayfs writes to upper layer with credentials of the user that mounted the overlay (usually root user).